src/filename_inspector.py

```diff
@@ -0,0 +1,106 @@
+import fnmatch
+import glob
+import os
+import json
+import csv
+
+
+def get_input(name: str) -> str:
+    value = os.getenv(f'INPUT_{name.upper()}')
+    return value
+
+
+def set_output(name: str, value: str):
+    output_file = os.getenv('GITHUB_OUTPUT', None)
+    if output_file:
+        with open(output_file, 'a') as f:
+            f.write(f'{name}={value}\n')
+    else:
+        raise EnvironmentError('GITHUB_OUTPUT environment variable is not set.')
+
+
+def set_failed(message: str):
+    print(f'::error::{message}')
+    exit(1)
+
+
+def info(message: str):
+    print(message)
+
+
+def find_non_matching_files(glob_patterns, suffix_patterns, exclude_patterns):
+    def matches_suffix(file, suffix_patterns):
+        for pattern in suffix_patterns:
+            if fnmatch.fnmatch(file, pattern):
+                return True
+        return False
+
+    def matches_exclude(file, exclude_patterns):
```

Comment:
Same as the above

Reply:
Fixed in commit 994f31093fbfb9e47abced9f7b8d1ad2d4235dfc.